makefile

```diff
@@ -4,6 +4,7 @@
 .SUFFIXES: .cpp .o .c .h
 
 CFLAGS = -fPIC -std=c89 -O3 -march=native -Wall -Wextra -pedantic
+CCFLAGS = -fPIC -O3 -march=native -Wall -Wextra -pedantic
```

Comment:
The file is C++, and GCC refused to compile C++ files with -std=c89.
But the correct macro would be CXXFLAGS, not CCFLAGS (which I used in my
patch). I will fix this.

2015-04-15 13:34 GMT+02:00 Anatol Belski <notifications@github.com>:

> In makefile
> <https://github.com/lemire/simdcomp/pull/7#discussion_r28411827>:
>
> > @@ -4,6 +4,7 @@
> >  .SUFFIXES: .cpp .o .c .h
> >
> >  CFLAGS = -fPIC -std=c89 -O3 -march=native -Wall -Wextra -pedantic
> > +CCFLAGS = -fPIC -O3 -march=native -Wall -Wextra -pedantic
>
> why did it need to remove C89 compat?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lemire/simdcomp/pull/7/files#r28411827>.
>